### ISSUE TITLE
>fm last (week/month/year) command doesn't work

### ISSUE THREAD

**Issue by Maahxxx** [joinemm/miso-bot#21]
When running the command you get the error: " AttributeError: 'NoneType' object has no attribute 'find'", I'm guessing it's because Last.FM changed the report cards. Will this issue be fixed?

**Comment by joinemm** [comment]
yeah, it's because they changed the page so my scraping code doesn't work anymore. I'll have to get to it but it will be fixed soon.
